vendor/terraform-0.11/APKBUILD

```diff
@@ -0,0 +1,32 @@
+# Contributor: Cloud Posse, LLC <hello@cloudposse.com>
+# Maintainer: Cloud Posse, LLC <hello@cloudposse.com>
+# vim: filetype=sh
+
+pkgver="${PACKAGE_VERSION}"
+pkgrel="${PACKAGE_RELEASE}"
+mjrver=${pkgver%.*}
+pkgname="terraform_${mjrver}"
+pkgdesc="${PACKAGE_DESCRIPTION}"
+arch="x86_64"
+url="https://releases.hashicorp.com/terraform/${pkgver}/terraform_${pkgver}_linux_amd64.zip"
+license="MPL-2.0"
+depends="dpkg"
+makedepends=""
+source="https://releases.hashicorp.com/terraform/${pkgver}/terraform_${pkgver}_linux_amd64.zip"
```

Comment:
`source` is defined but not used?

Reply:
This is not a shell script, but an alpine package configuration file. 

See https://wiki.alpinelinux.org/wiki/Creating_an_Alpine_package#source